operation-handbook.code-workspace

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is it normal practice to check in a file like this into source control?

Reply:
I've added it to the gitignore and will remove it. That was an oversight.

Reply:
removed!